ably/types/options.py

```diff
@@ -206,6 +207,10 @@ def loop(self):
     def auto_connect(self):
         return self.__auto_connect
 
+    @property
```

Comment:
Is it possible to just use the `Defaults` class when building the websocket URL? By putting `protocol_version` in the `Options` class it looks kinda like it's configurable, which it shouldn't be

Reply:
Yes, thought about that but I decided to put in `Options` incase a user wants to have that information. I left it out of the __init__ params, so users won't be able to configure it.

Reply:
I still think it's better to keep it out of `Options` tbh, users shouldn't need to know/care about protocol version and it's simpler if `Options` just contains user configuration and nothing else

Reply:
That makes perfect sense